### PR TITLE
Add calico-node -init command to mount bpffs.

### PR DESCRIPTION
**Pull request by fasaxc**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pkg/nodeinit/calico-init_windows.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodeinit
+
+const bpfFSMountPoint = "/sys/fs/bpf"
```
Comment (neiljerram) [inline pkg/nodeinit/calico-init_windows.go:17]: Is this needed on Windows?

File: pkg/nodeinit/calico-init_linux.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const bpfFSMountPoint = "/sys/fs/bpf"
+
+func Run() {
+	// Check $CALICO_STARTUP_LOGLEVEL to capture early log statements
+	startup.ConfigureLogging()
+
+	err := ensureBPFFilesystem()
+	if err != nil {
+		logrus.WithError(err).Error("Failed to mount BPF filesystem.")
+		os.Exit(2)
```
Comment (neiljerram) [inline pkg/nodeinit/calico-init_linux.go:40]: Does 2 have a special meaning here?  If so, maybe worth a comment to explain that.

**Review by neiljerram — APPROVED** [review]
LGTM, just a couple of nits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by fasaxc** [comment]
Tests passed but the result isn't showing up for some reason

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**